test/e2e/bundle_e2e_test.go

```diff
@@ -34,7 +105,7 @@ var _ = Describe("Installing bundles with new object types", func() {
 	})
 
 	When("a bundle with a pdb, priorityclass, and VPA object is installed", func() {
-		By("including the VPA CRD in the CSV")
+		By("first installing the VPA CRD on cluster")
```

Comment:
This call to By should be inside the BeforeEach so that it happens at test runtime instead of suite load time.